controllers/users.js

```diff
@@ -9,7 +9,10 @@ router.get('/', async function(req, res, next) {
     try {
         let users = await api(req).get('/users/', {
             qs: {
-                schoolId: req.query.schoolId
+                schoolId: req.query.schoolId,
+                roles: ['teacher'],
```

Comment:
Maybe we want to name it differently teachersOfSchool.js instead of users.js?
If you think about performance it's not nice to load 10.000 user objects (students/teacher of the complete school cloud) if a DB request can limit the effort to 50 users (teacher) of a defined school